examples/blaze/src/main/scala/com/example/http4s/blaze/demo/client/MultipartClient.scala

```diff
@@ -0,0 +1,43 @@
+package com.example.http4s.blaze.demo.client
+
+import cats.effect.{Effect, IO}
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import fs2.StreamApp.ExitCode
+import fs2.{Scheduler, Stream, StreamApp}
+import org.http4s.client.blaze.Http1Client
+import org.http4s.headers.`Content-Type`
+import org.http4s.multipart.{Multipart, Part}
+import org.http4s.{MediaType, Method, Request, Uri}
+
+object MultipartClient extends MultipartHttpClient[IO]
+
+class MultipartHttpClient[F[_]](implicit F: Effect[F]) extends StreamApp {
+
+  private val rick = getClass.getResource("/beerbottle.png")
+
+  private val multipart = Multipart[F](
+    Vector(
+      Part.formData("name", "gvolpe"),
+      Part.fileData("rick", rick, `Content-Type`(MediaType.`image/png`))
+    )
+  )
+
+  private val request =
+    Request[F](method = Method.POST, uri = Uri.uri("http://localhost:8080/v1/multipart"))
```

Comment:
Do we want to encourage people to use the `POST(uri, multipart)` syntax from `org.http4s.client.dsl`, or is that a failed experiment?

Reply:
I think I'm used to using it this way but good point, I'll change it in favor of the dsl version.

Reply:
Any ideas how we can do better here?  The ergonomics of request construction are one of our weak points.